operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md

```diff
@@ -0,0 +1,389 @@
+## Updating the Liquid-Cooled EX Cabinet CEC with Default Credentials after a CEC Password Change
+
+This procedure changes the credential for liquid-cooled EX cabinet chassis controllers and node controller (BMCs) used by CSM services after the CECs have been set to a new global default credential. 
+
+**NOTE:** This procedure does not provision Slingshot switch BMCs (RouterBMCs). Slingshot switch BMC default credentials must be changed using the procedures in the Slingshot product documentation. To update Slingshot switch BMCs, refer to "Change Rosetta Login and Redfish API Credentials" in the *Slingshot Operations Guide* (1.6.0).  
+
+This procedure provisions only the default Redfish root account passwords. It does not modify Redfish accounts that have been added after an initial system installation.
```

Comment:
@jimostrom-hpe The CSM procedure does not provision the redfish root account passwords (unless you count the stateful reset, but that is more the removal of existing creds and letting the CEC cred take presendence).

The CSM procedure just informs CSM of the that the Node and Chassis BMCs have be given new creds.